inc/zoo/swar/SWAR.h

```diff
@@ -77,18 +77,17 @@ struct SWAR {
 
     constexpr T value() const noexcept { return m_v; }
 
-    template<std::size_t N>
-    constexpr static T baseFromLaneLiterals(const T(&args)[N]) {
-        static_assert(N == Lanes, "Wrong number of lanes");
-        T result = 0;
+    template<std::size_t N, typename = std::enable_if_t<N == Lanes, T>>
```

Comment:
No.
You changed the arity of the template.
```suggestion
template<std::size_t N>
std::enable_if<N == Lanes, T>
baseFromLaneLiterals(const T (&args)[N]) {
```

Reply:
forgot the `constexpr static`

Reply:
ok will try and get this going!